gitindex/tree.go

```diff
@@ -99,7 +99,7 @@ func TreeToFiles(r *git.Repository, t *object.Tree,
 	rw := newRepoWalker(r, repoURL, repoCache)
 
 	if err := rw.parseModuleMap(t); err != nil {
-		return nil, nil, err
+		log.Printf("parseModuleMap: %s, ignoring error", err)
```

Comment:
I think somewhere we disable submodule support in our zoekt invocation. Can we maybe just skip these steps in that case?